modules/deploy/pages/deployment-option/cloud/managed-connectors/create-mongodb-source-connector.adoc

```diff
@@ -114,6 +114,10 @@ After the connector is created, check to ensure that:
 * There are no errors in logs and in Redpanda Console.
 * Redpanda topics contain data from relational database tables.
 
+== Use the Connectors API
+When using the Connectors API, instead of specifying a value for `connection.url`, `connection.username` and `connection.password`, 
+you can specify a value for `connection.uri`, of the form `mongodb+srv://username:password@cluster0.xxx.mongodb.net`.
```

Comment:
```suggestion
When using the Connectors API, instead of specifying a value for `connection.url`, `connection.username`, and `connection.password`, 
you can specify a value for `connection.uri` in the form `mongodb+srv://username:password@cluster0.xxx.mongodb.net`.
```